README.md

```diff
@@ -145,7 +153,9 @@ Chrome is pre-installed so that Django's selenium tests can be run in headless
 mode with a virtual display (id `:99`). For example, you may run a specific test
 like so:
 
-    (vm) $ runtests27-sqlite3 admin_widgets --selenium chrome
+    (vm) $ runtests36-sqlite3 admin_widgets --selenium chrome --parallel 1
+
+The test suite will sometimes hang when running selenium tests in parallel mode.
```

Comment:
No, I don't remember seeing this on the 1.11 branch. I'd planned to go back and test to see if we needed to open a ticket against Django.

Did you reproduce on master with this box, or another setup?